grails-app/views/wps/jobComplete.gsp

```diff
@@ -0,0 +1,11 @@
+Hi,
+
+Your IMOS download is ready. You can download it by clicking here: ${jobReportUrl}
```

Comment:
@pblain, this wording (or the original here) doesn't really make sense if the subset failed (because there will be nothing to download).

It needs to be something more like:

> Your IMOS subset has completed.  Access the results here:

or something better, but that leaves open the possibility of failure...

But if you understand all that and https://github.com/aodn/aodn-portal/pull/1995/files#r40636889 is definitely what you want, no problems!